src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -1,65 +1,204 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import { connect } from 'react-redux';
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
 import classNames from 'classnames';
 import recordEvent from 'platform/monitoring/record-event';
+import debounce from 'platform/utilities/data/debounce';
```

Comment:
They have a home rolled debounce func. I wish I knew that a couple of weeks ago haha

Reply:
hahah, gotta love finding things like this randomly